docs/usage/help.rst.inc

```diff
@@ -167,6 +167,16 @@ Examples::
     a directory, it won't recurse into it and won't discover any potential matches for
     include rules below that directory.
 
+    .. note::
+
+        It's possible that a sub-directory/file is matched while parent directories are not.
+        In that case, parent directories are not backed up thus their user, group, permission,
+        etc. can not be restored.
+
+        For example, with `--pattern=+/home/user/subdir --pattern=-/home/user/*`,
+        `/home/user/subdir` can be restored with correct metadata, while `\home`and
+        `/home/user` can not.
```

Comment:
can this be simplified?

also: no backslashes.

Reply:
I didn't get that we don't need to mention the special case in parent docs in another comment. Just removed this example to avoid complicating the doc.

Reply:
ok